web/pgAdmin4.py

```diff
@@ -13,7 +13,7 @@
 
 import sys
 if sys.version_info <= (3, 9):
-    import select
+    pass
```

Comment:
Does this change work with Python 3.9?

Reply:
No, I don't think so, I missed the requirements